opencti-platform/opencti-front/src/utils/hooks/useOverviewLayoutCustomization.ts

```diff
@@ -0,0 +1,41 @@
+type HalfWidth = 6;
+type FullWidth = 12;
+type OverviewLayoutCustomizationSettingsConfigurationWidth = HalfWidth | FullWidth;
+
+export interface OverviewLayoutCustomizationSettingsConfigurationParameters {
+  order: number;
+  width: OverviewLayoutCustomizationSettingsConfigurationWidth;
+}
+
+type OverviewLayoutCustomizationSettingsConfigurationWidgets = Map<string, OverviewLayoutCustomizationSettingsConfigurationParameters>;
+
+const threatActorIndividualConfiguration = new Map<string, OverviewLayoutCustomizationSettingsConfigurationParameters>();
```

Comment:
Don't think you should use a Map as GraphQL won't be able to provide one, you should rely on a simple Object I think

Reply:
The schema is exposed with an object with maps, in the front.
![image](https://github.com/user-attachments/assets/44eb4221-a2ae-465c-8f87-80ebaee869d0)

Reply:
I will do the same with the overview layout customization

Reply:
Passing an object would have complicated the access to a configuration based on the entity type sent to API.
=> 'Threat-Actor-Individual' can not be used as an object index. Whereas it can be used as is as a map key.